locales/de.json

```diff
@@ -0,0 +1,623 @@
+{
+  "lang": "de",
+  "rules": {
+    "accesskeys": {
+      "description": "",
+      "help": "Der Wert des accesskey-Attributes muss einzigartig sein."
+    },
+    "area-alt": {
+      "description": "",
+      "help": "Aktive <area>-Elemente müssen einen Alternativtext besitzen."
+    },
+    "aria-allowed-attr": {
+      "description": "",
+      "help": "Elemente dürfen nur erlaubte ARIA-Attribute verwenden."
+    },
+    "aria-dpub-role-fallback": {
+    "description": "",
+    "help": "Nicht unterstützte DPUB ARIA-Rollen sollten in Elementen mit impliziten Fallback-Rollen verwendet werden."
+    },
+    "aria-hidden-body": {
+      "description": "",
+      "help": "Aria-hidden='true' darf nicht für den <body> des Dokumentes verwendet werden."
+    },
+    "aria-required-attr": {
+      "description": "",
+      "help": "Erforderliche ARIA-Attribute müssen bereitgestellt werden."
```

Comment:
Erförderliche?

Reply:
"Erforderliche" is the correct translation for "required" in this case :-)